process_request/helpers.py

```diff
@@ -434,12 +435,11 @@ def identifier_from_uri(uri):
 def resolve_ref_id(repo_id, ref_id, client):
     """ Accepts options to find archival objects using find_by_id method.
 
-    Generates and returns a DIMES id from an ArchiveSpace URI.
+    Generates and returns a DIMES URI from an ArchiveSpace URI.
```

Comment:
By changing this return from an "id" to a URI, we gain a leading slash.  The caller in `LinkResolverView` in `process_request/views.py` is the only current caller, and needs to be updated.